packages/backend/src/telemetry/service.ts

```diff
@@ -94,10 +102,63 @@ class TelemetryServiceImpl implements TelemetryService {
     })
   }
 
+  public async incrementCounterWithTransactionFeeAmount(
+    name: string,
+    amountSource: { value: bigint; assetCode: string; assetScale: number },
+    amountDestination: { value: bigint; assetCode: string; assetScale: number },
+    attributes: Record<string, unknown> = {},
+    preservePrivacy = true
```

Comment:
I thought this was `false` by default when collecting fees metrics? 
```suggestion
    preservePrivacy = false
```